tests/converters/PGNConverterTests.cpp

```diff
@@ -0,0 +1,15 @@
+#include <gtest/gtest.h>
+#include "datastructures/Board.h"
+#include "converters/PGNConverter.h"
+
+TEST(PGNConverterTest, HandlesEmptyPGN) {
+    Board *testBoard = new Board(WHITE);
+    PGNConverter::convert("", testBoard);
+
+    ASSERT_EQ(1, 1);
+}
+
+int main(int argc, char **argv) {
+    ::testing::InitGoogleTest(&argc, argv);
```

Comment:
How would we go about running all tests?

Reply:
instead of `./stocklion` in the build steps you'd do `./stocklion_tests`